comp/core/workloadmeta/collectors/internal/kubeapiserver/kubeapiserver_test.go

```diff
@@ -152,7 +152,36 @@ func Test_metadataCollectionGVRs_WithFunctionalDiscovery(t *testing.T) {
 			expectedGVRs: []schema.GroupVersionResource{{Resource: "deployments", Group: "apps", Version: "v1"}},
 			cfg: map[string]interface{}{
 				"cluster_agent.kube_metadata_collection.enabled":   true,
-				"cluster_agent.kube_metadata_collection.resources": "deployments",
+				"cluster_agent.kube_metadata_collection.resources": "apps/deployments",
+			},
+		},
+		{
+			name: "only one resource (deployments), only one version, correct resource requested, but version is empty (with double slash)",
+			apiServerResourceList: []*metav1.APIResourceList{
+				{
+					GroupVersion: "apps/v1",
+					APIResources: []metav1.APIResource{
+						{
+							Name:       "deployments",
+							Kind:       "Deployment",
+							Namespaced: true,
+						},
+					},
+				},
+			},
+			expectedGVRs: []schema.GroupVersionResource{{Resource: "deployments", Group: "apps", Version: "v1"}},
+			cfg: map[string]interface{}{
+				"cluster_agent.kube_metadata_collection.enabled":   true,
+				"cluster_agent.kube_metadata_collection.resources": "apps//deployments",
```

Comment:
This case with double slash is no included in the examples that you added in the config.go file.

Reply:
I thought [this](https://github.com/DataDog/datadog-agent/blob/bfe0aefbf1d1406d960e58140502d2d705d74d82/pkg/config/setup/config.go#L478) line made it clear by saying `the version is optional and can be left empty`

I can add it as an example if you think it is not obvious.